package.json

```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "precommit": "lint-staged",
-    "preinstall": "node -e \"$(curl -fsSL https://raw.githubusercontent.com/Skyscanner/ensure-node-env/master/dist/index.js)\"",
+    "preinstall": "(curl -fsSLo checker.js https://raw.githubusercontent.com/Skyscanner/ensure-node-env/master/dist/index.js) && node checker.js && rm checker.js",
```

Comment:
I like this approach, but the problem with it is that if `node checker.js` fails, for example if you're on the wrong version of Node, `checker.js` won't actually be deleted.

If you change the final part to `&& node checker.js; rm checker.js` it should be deleted every time.